libraries/AP_Logger/LogStructure.h

```diff
@@ -1370,9 +1384,10 @@ struct PACKED log_PSCZ {
 // @Field: Safety: Hardware Safety Switch status
 
 // @LoggerMessage: PRX
-// @Description: Proximity sensor data
+// @Description: Proximity Filtered sensor data
```

Comment:
```suggestion
// @Description: Filtered Proximity sensor data
```

Reply:
I'm ok with leaving Proximity at the beginning.  I know it ends up with slightly odd english but it is good to have the subsystem at the front.

Reply:
Yes that was done on purpose. I remembered Randy preferred it this way..